strapi/strapi
# feat(content-releases): remove cron when scheduled release is deleted

### What does it do?

Remove the cron job when a scheduled release is delete. 

## Testing

### Enabling the future flags

The scheduling functionality is implemented through future flags. To begin, ensure the future flag is set to true. Navigate to `config/features.js` and include `contentReleasesScheduling` as an option within your future object. Create the future object if it does not exist.

Example:
```js
// config/features.js

module.exports = ({ env }) => ({
  future: {
    contentReleasesScheduling: true,
  },
});

```

### How to test it

Note: You need to get all the scheduledJobs from scheduling service (There is no a way in UI to see this, so you would need to go to the console)

1. Create a release with a scheduled date.
2. scheduledJobs should have the job for publish the just created release.
3. Delete the release
4. scheduledJobs should not have the job for publish the deleted release
